workshop/exercise-2/README.md

```diff
@@ -63,4 +63,119 @@ Before your continue, make sure all the pods are deployed and **`Running`**. If
 
 Congratulations! You successfully installed Istio into your cluster.
 
+
+## Validating Installation
+
+In this exercise, you will validate your Istio installation and launch some debugging containers to experiment with the service mesh.
+
+
+## Service Mesh Sidecar
+
+![istio architecture](istio-whatisit.svg)
+
+Istio's core functionality is achieved by setting up a proxy, or sidecar, inside every kubernetes pod. Outgoing and incoming traffic bound for the pod is intercepted by iptables and delivered to the `envoy` daemon running in the sidecar. The `envoy` proxy can then inspect, modify, forward, and record all traffic used by the application. East-West traffic goes through two `envoy` proxies, one at source and one at destination. All `envoy` proxies are configured by `pilot`, an Istio component.
+
```

Comment:
would be great to add some hyperlinks to envoy and pilot.

Reply:
It seems incomplete to describe other key control plane components here, citadel, telemtry and policy.

Reply:
Will fix